bentoml/deployment/sagemaker/__init__.py

```diff
@@ -57,29 +57,15 @@
 
 EXPOSE 8080
 
-RUN set -x \\
-     && apt-get update \\
-     && apt-get install --no-install-recommends --no-install-suggests -y libpq-dev build-essential\\
-     && apt-get install -y nginx \\
-     && rm -rf /var/lib/apt/lists/*
-
-# pre-install BentoML base dependencies
-RUN conda install pip numpy scipy \\
-      && pip install gunicorn gevent
+RUN apt-get update --fix-missing && \
+    apt-get install -y nginx && \
+    apt-get clean
 
 # copy over model files
 COPY . /opt/program
 WORKDIR /opt/program
 
-# update conda base env
-RUN conda env update -n base -f /opt/program/environment.yml
-RUN pip install -r /opt/program/requirements.txt
-
-# Install additional pip dependencies inside bundled_pip_dependencies dir
-RUN if [ -f /bento/bentoml_init.sh ]; then /bin/bash -c /bento/bentoml_init.sh; fi
```

Comment:
@yubozhao  it looks like this code was problematic - the bundled bentoml was never installed properly for sagemaker deployment